sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Microsoft.Azure.Management.Attestation.Tests.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>Attestation.Tests</PackageId>
+    <Version>1.0.0</Version>
+    <Description>Attestation.Tests Class library</Description>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
@shleiAmy  TargetFrameworks is already defined in AzSdk.test.reference.props, please remove line 8

Reply:
removed  <TargetFrameworks>netcoreapp2.0</TargetFrameworks>